chamge/amqp-hub-backend.c

```diff
@@ -104,6 +104,43 @@ _amqp_rpc_reply_string (amqp_rpc_reply_t r)
   }
 }
 
+inline static const gchar *
+_amqp_get_rpc_reply_string (amqp_rpc_reply_t r)
```

Comment:
I see we're duplicating a lot of code between the different AMQP backends. There's definitely a lot to be improved in the matter of maintainability of Chamge code. I know we're about to replace rabbitmq with another library; it's just something to have in mind when we're re-implementing this.

Reply:
Yes, it is the same issue I pointed but @justinjoy said

> It has some reason to have duplicate code in amqp-*-backend;
> 
>  * we didn't sure librabbitmq-c is stable or production-ready
>  *  we tended to implement each backend by a different developer, and different style to test librabbitmq-c in various ways while implementation
>  *  but, finally, lack of time and developer. As a result, we do just copy-n-paste
> 
> Having a common set is good idea, but now modifying chamge-arbiter and chamge-edge to have a common function set is a very low priority.
> One option to raise priority the task is only if you find a severe bug while handling chamge-hub APIs.